lib/versioning/cargo/index.ts

```diff
@@ -9,7 +10,12 @@ export const urls = [
   'https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html',
 ];
 export const supportsRanges = true;
-export const supportedRangeStrategies = ['bump', 'extend', 'pin', 'replace'];
+export const supportedRangeStrategies: RangeStrategy[] = [
+  'bump',
+  'widen',
+  'pin',
+  'replace',
+] as RangeStrategy[];
```

Comment:
```suggestion
];
```

You need to revert all the other `as` statements too as they result in the arrays *not* being validated.

Reply:
@rarkins Whitout this compilation fails 🤔

Reply:
Strange. But with it, there's no type validation?

Reply:
Not sure. I've committed your suggestion to check ci.